php/ClassInfo/ReflectionClass.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace PHPCD\ClassInfo;
+
+use PHPCD\PatternMatcher\PatternMatcher;
+
+class ReflectionClass extends \ReflectionClass implements ClassInfo
+{
+    /**
+     * @var PatternMatcher
+     */
+    private $pattern_matcher;
+
+    /**
+     * @param string|object $class
+     * @param PatternMatcher $pattern_matcher
+     */
+    public function __construct($class, PatternMatcher $pattern_matcher)
+    {
+        parent::__construct($class);
```

Comment:
compose is better than inherit. Why not let ReflectionClass to hod a instance of the `\ReflectionClass`?